homeassistant/components/hassio/strings.json

```
@@ -274,7 +274,7 @@
       "fields": {
         "addon": {
           "name": "Add-on",
-          "description": "The add-on slug."
+          "description": "The add-on for this action."
```

This sounds strange, because it does not really explain something about the field, would it make sense to remove the references to the description from the other fields and just change this to
```suggestion
          "description": "The add-on to start."
```